sns_proposal.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+if [ -z "${IC_ADMIN}" ]; then
+  echo "No IC_ADMIN set"
+  exit 1
+fi
+
+if [ -z "${NEURON_ID}" ]; then
+    echo "NEURON_ID not set"
+    exit 1
+fi
+
+check_or_set_dfx_hsm_pin() {
+    VALUE=${DFX_HSM_PIN:-}
+    if [ -z "$VALUE" ]; then
+        echo -n "Enter your HSM_PIN":
+        read -s DFX_HSM_PIN
+        export DFX_HSM_PIN
+        echo
+    fi
+}
+
+PROPOSAL_TITLE="Proposal to create an SNS-DAO for OpenChat"
+PATH_TO_SUMMARY_FILE=$PWD/summary_file.md
+
+SUMMARY=`cat $PATH_TO_SUMMARY_FILE`
+
+check_or_set_dfx_hsm_pin
+
+$IC_ADMIN --use-hsm --slot=0 --key-id=01 --pin=$DFX_HSM_PIN \
+              --nns-url https://nns.ic0.app \
+              propose-to-open-sns-token-swap \
+              --proposer $NEURON_ID \
+              --min-participants 100 \
+              --min-icp-e8s 50000000000000 \
+              --max-icp-e8s 100000000000000 \
+              --community_fund_investment_e8s 33333300000000 \
+              --min-participant-icp-e8s 100000000 \
+              --max-participant-icp-e8s 15000000000000 \
+              --swap-due-timestamp-seconds 1209600 \
+              --sns-token-e8s 2500000000000000 \
+              --target-swap-canister-id zcdfx-6iaaa-aaaaq-aaagq-cai \
```

Comment:
What's this canisterId? A test SNS canister?

Reply:
Maybe we should say <SNS_SWAP_CANISTER_ID> so that we know we need to update it once we have deployed the SNS.

Reply:
Probably SNS1. But doesn't matter for now